yt_dlp/extractor/playsuisse.py

```diff
@@ -134,12 +135,35 @@ class PlaySuisseIE(InfoExtractor):
             id
             url
         }'''
+    _ID_TOKEN = None
+
+    def _perform_login(self, username, password):
+        login_page = self._download_webpage(
+            'https://www.playsuisse.ch/api/sso/login', None, note='Downloading login page',
+            query={'x': 'x', 'locale': 'de', 'redirectUrl': 'https://www.playsuisse.ch/'})
+        settings = self._search_json(r'var\s+SETTINGS\s*=', login_page, 'settings', None)
+
+        webpage, urlh = self._download_webpage_handle(
+            'https://login.srgssr.ch/srgssrlogin.onmicrosoft.com/B2C_1A__SignInV2/SelfAsserted',
+            None, note='Logging in', headers={'X-CSRF-TOKEN': settings['csrf']},
+            data=urlencode_postdata({'request_type': 'RESPONSE', 'signInName': username, 'password': password}),
+            query={'tx': settings['transId'], 'p': 'B2C_1A__SignInV2'})
+
+        webpage, urlh = self._download_webpage_handle(
+            'https://login.srgssr.ch/srgssrlogin.onmicrosoft.com/B2C_1A__SignInV2/api/CombinedSigninAndSignup/confirmed',
+            None, note='Downloading ID token',
+            query={'rememberMe': 'false', 'csrf_token': settings['csrf'], 'tx': settings['transId'], 'p': 'B2C_1A__SignInV2', 'diags': ''})
```

Comment:
we're not using `webpage` for anything, so `_request_webpage()` will suffice
```suggestion
        urlh = self._request_webpage(
            f'{self._LOGIN_BASE_URL}/{self._LOGIN_PATH}/api/CombinedSigninAndSignup/confirmed',
            None, 'Downloading ID token', query={
                'rememberMe': 'false',
                'csrf_token': csrf_token,
                **query,
                'diags': '',
            })
```